sdk/src/state_machine/stf.rs

```diff
@@ -56,18 +83,11 @@ pub trait StateTransitionFunction {
     fn begin_slot(&mut self);
 
     /// Apply a batch of transactions to the rollup, slashing the sequencer who proposed the batch on failure
-    fn apply_batch(
+    fn apply_blob(
         &mut self,
-        batch: Self::Batch,
-        sequencer: &[u8],
+        blob: impl BlobTransactionTrait,
```

Comment:
Why do we pass here `impl BlobTransactionTrait` why the other types are defined as `StateTransitionFunction`  type parameters?

Reply:
Let's add a comment why we use `impl BlobTransactionTrait`

Reply:
Done!